docs/source/experiment_logging.rst

```diff
@@ -1,201 +1,186 @@
 Experiment Logging
-===================
+==================
```

Comment:
not sure but I think that doctest in RST files are not executed, so there is no need of converting them here
https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html